src/components/application_manager/rpc_plugins/rc_rpc_plugin/test/commands/button_press_request_test.cc

```diff
@@ -168,6 +169,7 @@ class ButtonPressRequestTest
       mock_allocation_manager_;
   testing::NiceMock<rc_rpc_plugin_test::MockInteriorDataCache>
       mock_interior_data_cache_;
+  rc_rpc_plugin::RCRPCPlugin rcrpc_plugin_;
```

Comment:
Please do not use real classes ( dependencies) for testing of other classes. Ise mocks